api/routes.go

```diff
@@ -57,6 +57,9 @@ func addRoutes(r *gin.Engine, auth Authentication, tokenHandler TokenHandler, uc
 	router.POST("/decisions/all",
 		timeoutMiddleware(SEQUENTIAL_DECISION_TIMEOUT),
 		handlePostAllDecisions(uc, marbleAppHost))
+	// query in the form localhost:8085/decisions/databyoutcome?scenario_id='123-abc'&created_ad=1992-03-15&expires_at=1992-03-15"
+	router.GET("/decisions/databyoutcome", timeoutMiddleware(LIST_DECISION_TIMEOUT), handleDecisionsDataByOutcome(uc))
```

Comment:
two questions here on the API:
- would it make sense to have one endpoint that returns all the stats, both for real decisions and phantom decisions related to a given test run ? I'm open to both approaches and we can discuss it tomorrow, but I'm a bit afraid of API endpoint inflation (and they would always be used together)
- either way, and this is more of a detail, if the API exposes the before/after filters on decisions, my feeling is that they should be called accordingly (maybe "created_before"/"created_after"): created_at and expires_at, relating to the name of fields on the test run, do not by themselves have meaning in the context of an endpoint on decisions